wasm/tests/conftest.py

```diff
@@ -0,0 +1,108 @@
+import subprocess
+import os
+import time
+import socket
+import atexit
+import pytest
+import sys
+
+PORT = 8080
+
+HTTP_SCRIPT = f"""
+import mimetypes
+mimetypes.add_type("application/wasm", ".wasm")
+import http.server
+http.server.test(HandlerClass=http.server.SimpleHTTPRequestHandler, port={PORT})
+"""
+
+demo_dist = os.path.join(os.path.dirname(os.path.realpath(__file__)), "../demo/dist/")
+
+server_proc = None
+
+
+def pytest_sessionstart(session):
```

Comment:
How about changing this to be a module scope fixture?

Reply:
Then the server would have to be started and stopped for every test, and given that a simple http server is stateless anyway, I don't think it makes much of a difference.